### ISSUE TITLE
"object-curly-spacing" insconsistently applied in imports

### ISSUE THREAD

**Issue by garborg** [eslint/eslint#3302]
```js
/*eslint "object-curly-spacing": [2, "always"]*/
"use strict";

import { Router, Route } from "react-router"; // rule enforces this
import { b } from "a"; // rule enforces this
const { c } = b; // rule enforces this

import C, {d} from "c"; // should throw an error but doesn't

/* Related: I searched far and wide, through issues and
   'spac*'-, 'curly'-, and 'block*'- related rules, but I couldn't find it:
   Can I make the missing space inside this curly an error?
*/
function nop() {return; }

Router.x(Route.x(C.x(c, d, nop())));
```

To verify, clone [garborg/eslint-curly-test](https://github.com/garborg/eslint-curly-test) and `npm install && npm start`.

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by garborg** [comment]
eslint -v -> 1.0.0

eslint output -> no errors, missing errors are noted in comments in snippet above

**Comment by nzakas** [comment]
Maybe an issue with having multiple imports in one statement.

**Comment by xjamundx** [comment]
Will look into it today.

**Comment by garborg** [comment]
Appreciate the quick resolution, @xjamundx!
